manim/plugins/import_plugins.py

```diff
@@ -0,0 +1,44 @@
+import types
+from importlib import import_module
+
+import pkg_resources
+
+from .. import config, logger
+
+__all__ = []
+
+module_type = types.ModuleType
+function_type = types.FunctionType
```

Comment:
These variables shouldn't be necessary at all anymore and you can just use `types.ModuleType` and `types.FunctionType` in their place